src/MinHeap.test.js

```diff
@@ -0,0 +1,114 @@
+var MinHeap = require('./MinHeap');
+
+describe('MinHeap', function() {
+  describe('#getFirst', function() {
+    describe('when initialized', function() {
+      it('first value is undefined', function() {
+        var heap = MinHeap();
+        expect(heap.getFirst()).toBeUndefined();
+      });
+    });
+
+    describe('after initialization', function() {
+      describe('when a value is added', function() {
+        it('first value is added value', function() {
+          var heap = MinHeap();
+          heap.add(1);
+          expect(heap.getFirst()).toBe(1);
+        });
+
+        describe('when the value is removed', function() {
+          it('first value is undefined', function() {
+            var heap = MinHeap();
+            heap.add(1);
+            heap.remove();
+            expect(heap.getFirst()).toBeUndefined();
+          });
+        });
+      });
+    });
+  });
+
+  describe('#add', function() {
+    describe('when empty', function() {
+      it('value is first value', function() {
```

Comment:
This seems to be duplicated case then `after initialization when a value is added first value is added value`